examples/flutter_login/lib/profile/bloc/profile_bloc.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:hydrated_bloc/hydrated_bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:user_repository/user_repository.dart';
+
+part 'profile_event.dart';
+part 'profile_state.dart';
+
+class ProfileBloc extends HydratedBloc<ProfileEvent, ProfileState> {
```

Comment:
rather than introducing a `ProfileBloc` can we convert the `AuthenticationBloc` to a `HydratedBloc`?